src/display/api.js

```diff
@@ -951,17 +951,60 @@ var PDFPageProxy = (function PDFPageProxyClosure() {
     },
 
     /**
+     * @param {boolean} normalizeWhitespace - replaces all occurrences of
+     * whitespace with standard spaces (0x20). The default value is `false`.
+     * @param {boolean} disableCombineTextItems - do not attempt to combine
+     * same line {@link TextItem}'s. The default value is `false`.
```

Comment:
The method doesn't take parameters, but it takes an optional object, so the above comments should be replaced with what's used below:
`* @param {getTextContentParameters} params - getTextContent parameters.`